codegen/src/generator/mod.rs

```diff
@@ -115,7 +115,7 @@ fn generate<P, E>(writer: &mut FileWriter, service: &Service, protocol_generator
         use hyper::Client;
         use hyper::status::StatusCode;
         use request::DispatchSignedRequest;
-        use region;
+        // use rusoto::region;
```

Comment:
Turns out this change isn't needed, thus I didn't have to touch every service file to manually import it.  This was leftover from experimenting with importing `rusoto` piecemeal instead of `use rusoto;` in the S3 crate.